ffi/java-idsdk/src/test/java/com/iotics/sdk/identity/App.java

```diff
@@ -0,0 +1,75 @@
+package com.iotics.sdk.identity;
+
+import com.iotics.sdk.identity.experimental.JWT;
+import com.iotics.sdk.identity.jna.JnaSdkApiInitialiser;
+import com.iotics.sdk.identity.jna.SdkApi;
+
+import java.time.Duration;
+
+public class App {
+    static String resolver = "https://did.stg.iotics.com";
+    static String seed = "f25a09c9d21ad5f7535fac4c30afe1a9f2ca025a192db549044b1b0130d1e945";
+    static String uDiD = "did:iotics:iotEBuXp2wHMREZmwYAyPhFzPYfWtt9Ka2R2";
+    static Identity agentIdentity = new Identity("aKey1", "#app1", "did:iotics:iotJxn2AHBkaFXKkBymbFYcVokGhLShLtUf1");
+
+    public static void main(String[] args) {
+        SdkApi api = new JnaSdkApiInitialiser("../bin/lib-iotics-id-sdk-amd64.so").get();
```

Comment:
Path config? Other platforms?

Reply:
this is a sample app that one can use for it's own purposes - it's likely to be used in-situ by
1-compire the so from .. (make so)
2-run this App

so I choose to use hardcoded values as means of providing defaults for the most common cases
for other platforms the user may want to do a clever switch based on the current OS